lib/cdo/graphics/certificate_image.rb

```diff
@@ -169,7 +166,7 @@ def self.create_course_certificate_image(name, course=nil, donor_name=nil, cours
     end
 
     if default_random_donor && !donor_name
-      donor = CdoDonor.get_random_donor_by_weight
+      donor = PegasusCdoDonor.get_random_donor_by_weight
```

Comment:
hey @pablo-code-org , the challenge here is that certificate_image.rb is used by both pegasus and dashboard. Due to some quirks in the way we run pegasus locally and in drone, I have some concerns about this particular change sneaking through drone but then breaking in the DTT. If you want to proceed with this, I would suggest spinning up an adhoc and making sure the hour_of_code_finish.feature UI test is passing there.

alternately, what we had talked about at the end of last week was putting this change on hold until the congrats work is done, which I am planning to complete in the next few weeks.

Reply:
Oh, I'll leave the PR ready and wait for the image certificate changes